examples/streamz/Dockerfile

```diff
@@ -1,54 +1,57 @@
-ARG RAPIDS_VERSION=0.13
-ARG CUDA_VERSION=10.1
+# An integratio:n test & dev container which builds and installs CLX from default branch
+ARG CUDA_VERSION=10.2
+ARG CUDA_SHORT_VERSION=${CUDA_VERSION}
 ARG LINUX_VERSION=ubuntu18.04
 ARG PYTHON_VERSION=3.7
+FROM gpuci/miniconda-cuda:${CUDA_VERSION}-devel-${LINUX_VERSION}
```

Comment:
just fyi we can go back to rapidsai-dev-nightly image since blas requirements have been relaxed.